### PR TITLE
Update Tocilizumab section

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

I haven't edited the clinical trials section yet, but had a few questions for @aadattoli, so it seemed like I should just push what I have so far

### Related issues

#364 #75 #545 #464 

### Suggested reviewers (optional)

@aadattoli Ada was the original author of this section
@hufengling, If I recall you are super knowledgeable about clinical trials and methodology, would you be willing to give this a read?
@dziakj1 You also seem like someone who might know how to communicate the results of three metanalyses that are analyzing the same flawed studies!

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.treatments.md [hunk 1]
```diff
@@ -617,37 +617,33 @@ Biologics are produced from components of living organisms or viruses. They incl
...
+IL-6 synthesis is tightly regulated both transcriptionally and post-transcriptionally, and it has been shown that viral proteins can enhance transcription of the IL-6 gene by  strengthening the DNA-binding activity between several transcription factors and IL-6 gene-cis-regulatory elements [@doi:10.2492/inflammregen.33.054].
+Therefore, drugs inhibiting the binding of IL-6 to IL-6Rα or sIL-6Rα are of interest for combatting the hyperactive inflammatory response characteristic of the cytokine storm.
+TCZ is a humanized monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, providing de facto inhibition of the IL-6 immune cascade.
 
 **Current Evidence.**
-Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear signs of improvements, according to their results [@url:https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/].
+Tocilizumab is being investigated either as a primary or supplemental treatment in {{ebm_tocilizumab_ct}} COVID-19 clinical trials curently registered at clinicaltrials.gov.
```
Comment (rando2) [inline content/20.treatments.md:645]: @agitter, this was my guess of how to invoke the stored information from the json file -- but I wanted to run it by you!

File: content/20.treatments.md [hunk 2]
```diff
@@ -617,37 +617,33 @@ Biologics are produced from components of living organisms or viruses. They incl
...
+IL-6 synthesis is tightly regulated both transcriptionally and post-transcriptionally, and it has been shown that viral proteins can enhance transcription of the IL-6 gene by  strengthening the DNA-binding activity between several transcription factors and IL-6 gene-cis-regulatory elements [@doi:10.2492/inflammregen.33.054].
+Therefore, drugs inhibiting the binding of IL-6 to IL-6Rα or sIL-6Rα are of interest for combatting the hyperactive inflammatory response characteristic of the cytokine storm.
+TCZ is a humanized monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, providing de facto inhibition of the IL-6 immune cascade.
 
 **Current Evidence.**
-Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear signs of improvements, according to their results [@url:https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/].
+Tocilizumab is being administered either as a primary or supplemental treatment in {{ebm_tocilizumab_ct}} COVID-19 clinical trials curently registered at clinicaltrials.gov.
```
Comment (agitter) [inline content/20.treatments.md:645]:
```suggestion
Tocilizumab is being administered either as a primary or supplemental treatment in {{ebm_tocilizumab_ct}} COVID-19 clinical trials (Figure @fig:ebm-trials). 
```

Yep, you got the template variable syntax right.  This should populate the correct number after tonight's nightly external resources update 🤞

We can rephrase this to be more broad.  There are more sources than clinicaltrials.gov. The COVID-19 trials data source [says](http://covid19.trialstracker.net/about/)
> we get our trial data from the ICTRP which holds structured data from 18 individual trial registries including ClinicalTrials.gov

One idea would be to simply refer to the trials figure, which links to the original data source.  That's what I suggested here.

I don't know clinical trials terminology very well.  Does "primary or supplemental" cover trials where it is part of the standard of care?

**Comment by rando2** [comment]
Thank you so much for feedback @agitter and @aadattoli! I'm going to mark this as a draft because I have one or two more studies to add, but hopefully will get it finalized tomorrow AM!

File: content/20.treatments.md [hunk 3]
```diff
@@ -617,57 +617,117 @@ Biologics are produced from components of living organisms or viruses. They incl
...
-This aspect should be investigated further in upcoming trials.
-
-Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the COVID-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late.
+Tocilizumab is being administered either as a primary or supplemental treatment in {{ebm_tocilizumab_ct}} COVID-19 clinical trials curently registered at clinicaltrials.gov. 
+No randomized, placebo-controlled studies have been released yet for TCZ.
+Therefore, no conclusions can be drawn about its efficacy for the treatment of COVID-19.
+However, early interest in TCZ as a possible treatment for COVID-19 emerged from a very small retrospective study in China that examined 20 patients with severe symptoms in early February 2020 and reported rapid improvement in symptoms following treatment with TCZ [@doi:10.1073/pnas.2005615117].
+Subsequently, a number of restrospective studies have been conducted in several countries.
+<!--
```
Comment (rando2) [inline content/20.treatments.md:650]: I was going to add a table demonstrating some of the studies, but then I found that the most recent systematic review already did a better job...

**Comment by rando2** [comment]
There are definitely more papers we could incorporate, but I think this is a pretty representative sample of what's out there for now. Definitely open to criticism, I was planning to include that table and then found a lit review that had a better table...

**Comment by rando2** [comment]
Not sure what went wrong here, I thought I was working off an up-to-date branch but I just had to resolve a ton of conflicts!

File: content/02.introduction.md [hunk 4]
```diff
@@ -147,6 +147,20 @@ After a virus enters into a host cell, its antigen is presented by major histoco
 One of the main immune responses that contribute to the onset of Acute Respiratory Distress Syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
 In addition to respiratory distress, this mechanism leads to organ failure and death in severe COVID-19 cases [@doi:10.1016/j.jpha.2020.03.001].
 
+##### Cytokine Storm
+
+<!-- As I am cleaning up the text, I'll move the cytokine storm text here, as right now it's very decentralized and one of our goals is to merge everything together in the intro!-->
+IL-6 promotes the differentiation of activated B cells into immunoglobulin-producing plasma cells [@pmid:9052836] and acts as a growth factor for hybridoma and myeloma cells [@pmid:8431556; @pmid:2104241].
+In addition, IL-6 also induces the differentiation of naïve CD4+ T cells into effector T-cell subsets [@doi:10.1158/2326-6066.CIR-14-0022].
+In this way interleukins regulate both the pro- and anti-inflammatory responses.
```
Comment (ajlee21) [inline content/02.introduction.md:155]: I apologize if this is already somewhere else in the text, and I've missed it. Would it make sense to define pro vs anti-inflammatory response? Since the balance of these two is brought up as being important later.

File: content/02.introduction.md [hunk 5]
```diff
@@ -147,6 +147,20 @@ After a virus enters into a host cell, its antigen is presented by major histoco
...
+In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system, ensuring homeostasis.
+The introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-α [@doi:10.1016/j.virusres.2007.02.007], and deceased SARS-CoV patients were found to have intermediate levels of IL-6, IL-1𝛽, and TNF-α expressed in a number of ACE2-expressing cell types sampled from the lung and bronchial tissues during autopsy [@doi:10.1002/path.2067].
+However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor α (TNF-α) and IL-1𝛽 decreased with the onset of ARDS [@doi:10.1164/ajrccm.164.10.2104013].
```
Comment (ajlee21) [inline content/02.introduction.md:158]: Just to clarify, is the point here that SARS patients have intermediate levels of IL-6 but other respiratory conditions, like ARDS have higher levels of IL-6?

As a non-immunologist, my high level understanding of what is written is: IL-6 plays a role in regulating the immune response to infection. And different IL-6 levels are found depending on the disease. This IL-6 cytokine causes a large immune response that damages cells and is fatal.

File: content/20.treatments.md [hunk 6]
```diff
@@ -617,57 +617,117 @@ Biologics are produced from components of living organisms or viruses. They incl
...
+TCZ is a receptor antibody that was developed to manage chronic inflammation caused by the continuous synthesis of the cytokine interleukin-6 (IL-6) [@doi:10.1101/cshperspect.a016295].
+IL-6 is a pro-inflammatory cytokine belonging to the interleukin family, immune system regulators that are primarily responsible for immune cell differentiation.
+Often used to treat conditions such as rheumatoid arthritis [@doi:10.1101/cshperspect.a016295], TCZ has become a pharmaceutical of interest for the treatment of COVID-19. 
+While secretion of IL-6 can be associated with chronic conditions, it is a key player in the innate immune response that is secreted by macrophages in response to the detection of PAMPs or DAMPs [@doi:10.1101/cshperspect.a016295].
+An analysis of 191 in-patients at two Wuhan hospitals revealed that blood concentrations of IL-6 were elevated at admission in adult COVID-19 patients who ultimately deceased compared to those who survived and remained higher, on average, throughout the course of hospitalization [@doi:10/ggnxb3].
```
Comment (ajlee21) [inline content/20.treatments.md:624]: The second half of this sentence is a little confusing to me. So the IL-6 levels in those patients that were died were higher than those who survived?

File: content/20.treatments.md [hunk 7]
```diff
@@ -617,57 +617,117 @@ Biologics are produced from components of living organisms or viruses. They incl
...
+Unlike membrane-bound IL-6Rα, which is only found on hepatocytes and some types of leukocytes, gp130 is found on most cells [@doi:10.1007/s00430-006-0019-9].
+When IL-6 binds to sIL-6Rα, the complex can then bind to a gp130 protein on any cell [@doi:10.1007/s00430-006-0019-9].
+The binding of IL-6 to IL-6Rα is termed classical signaling, while its binding to sIL-6Rα is termed trans-signaling [@doi:10.1016/j.cytogfr.2012.04.001; @doi:10.1042/bj3000281; @doi:10.1007/s00430-006-0019-9].
+These two signaling processes are thought to play different roles in health and illness; for example, trans-signaling may play a role in the proliferation of mucosal T-helper TH2 cells associated with asthma, while an earlier step in this proliferation of process, the CD4+-cells towards the TH2 lineage, may be regulated by classical signaling, and IL-6 is known to play a role in Crohn's Disease via trans-, but not classical, signaling [@doi:10.1007/s00430-006-0019-9].
```
Comment (ajlee21) [inline content/20.treatments.md:636]: Maybe break up this sentence

File: content/20.treatments.md [hunk 8]
```diff
@@ -617,57 +617,117 @@ Biologics are produced from components of living organisms or viruses. They incl
...
+
+Under this definition, SOC could include other drugs such as HCQ, azithromycin, lopinavir-ritonavir or darunavir-cobicistat, or heparin.
+While this study was not randomized, a subset of patients who were eligible to receive TCZ were unable to obtain it due to shortages; however, these groups were not directly compared in the analysis.
+After adjusting for variables such as age, sex, and SOFA score, they found that patients treated with TCZ were less likely to progress to invasive medical ventilation and/or death (adjusted hazard ratio = 0.61, CI 0.40-0.92, p=0.020), although analysis of death and ventilation separately suggests that this effect may have been driven by differences in the death rate (20% of control versus 7% of TCZ-treated patients).
+They reported particular benefits for patients whose P/F Ratio fell below a 150 mm Hg threshold and found no differences between groups administered subcutaneous versus intravenous TCZ.
```
Comment (ajlee21) [inline content/20.treatments.md:677]: What is the P/F ratio?

File: content/20.treatments.md [hunk 9]
```diff
@@ -617,57 +617,117 @@ Biologics are produced from components of living organisms or viruses. They incl
...
+Another retrospective observational analysis of interest examined the charts of patients at a hospital in Connecticut, USA where 64% of all 239 COVID-19 patients in the study period were administered TCZ based on assignment by a standardized algorithm [@doi:10.1016/j.chest.2020.06.006].
+They found that survivorship was more similar in patients presenting with severe vs. nonsevere COVID-19  at intake (severe vs. nonsevere) when TCZ was administered and proposed that their algorithm was able to target TCZ to patients presenting with CRS.
```
Comment (ajlee21) [inline content/20.treatments.md:679]: A little confused by this sentence. So patients with severe or nonsevere covid had similar survival when treated with TCZ?

File: content/20.treatments.md [hunk 10]
```diff
@@ -617,57 +617,117 @@ Biologics are produced from components of living organisms or viruses. They incl
...
+They also reported higher survivorship in Black and Hispanic patients compared to white patients when adjusted for age.
+The major limitation with interpretation for these studies is that there may be clinical characteristics that influenced medical practitioners decisions to administer TCZ to some patients and not others.
+One interesting example therefore comes from an analysis of patients at a single hospital in Brescia, Italy, where TCZ was not available for a period of time [@doi:10.1016/j.ejim.2020.05.009].
+This study compared COVID-19 patients admitted to the hospital before and after March 13, 2020, when the hospital received TCZ.
```
Comment (ajlee21) [inline content/20.treatments.md:683]: Was the characteristics of the patients similar before vs after? i.e. similar age, gender, severity of symptoms?

File: content/20.treatments.md (hunk 9 above)
Comment (rando2) [reply content/20.treatments.md:679]:
```suggestion
They found that TCZ administration was associated with more similar rates of survivorship in patients with severe versus nonsevere COVID-19 at intake, and they therefore proposed that their algorithm was able to identify patients presenting with CRS as good candidates for TCZ.
```
That is what they reported -- in the patients who got TCZ, their severity at intake was less strongly associated with whether they passed away.

File: content/20.treatments.md (hunk 10 above)
Comment (rando2) [reply content/20.treatments.md:683]: Great question Alex. So they didn't do statistical tests but they don't look comparable! That is a great catch. See comment below.

File: content/20.treatments.md [hunk 11]
```diff
@@ -617,57 +617,117 @@ Biologics are produced from components of living organisms or viruses. They incl
...
+Therefore, patients who would have been eligible for TCZ prior to this arbitrary date did not receive it as treatment, making this retrospective analysis something of a natural experiment.
```
Comment (rando2) [inline content/20.treatments.md:684]:
```suggestion
Therefore, patients who would have been eligible for TCZ prior to this arbitrary date did not receive it as treatment, making this retrospective analysis something of a natural experiment.
Despite this design, demographic factors did not appear to be consistent between the two groups, and the average age of the control group was older than the TCZ group.
The control group also had a higher percentage of males and a higher incidence of comorbidities such as diabetes and heart disease.
All the same, the multivariate hazard ratio, which adjusted for these clinical and demographic factors, found a significant difference between survival in the two groups (HR=0.035, CI=0.004-0.347, p = 0.004).
```

File: content/20.treatments.md (hunk 6 above)
Comment (ajlee21) [reply content/20.treatments.md:624]:
```suggestion
An analysis of 191 in-patients at two Wuhan hospitals revealed that blood concentrations of IL-6 were elevated at admission in adult COVID-19 patients, who ultimately deceased, compared to those who survived[@doi:10/ggnxb3].

IL-6 levels remained higher on average, in those patients, who ultimately deceased, through the course of their hospitalization [@doi:10/ggnxb3].
```

File: content/20.treatments.md (hunk 9 above)
Comment (ajlee21) [reply content/20.treatments.md:682]: Just to clarify, "severe patients" are referring to those with CRS?

File: content/20.treatments.md (hunk 6 above)
Comment (rando2) [reply content/20.treatments.md:624]:
```suggestion
An analysis of 191 in-patients at two Wuhan hospitals revealed that blood concentrations of IL-6 differed between patients who did and did not recover from COVID-19.
Patients who ultimately deceased had higher IL-6 levels at admission than those who recovered [@doi:10/ggnxb3].
Additionally, IL-6 levels remained higher throughout the course of hospitalization in the patients who ultimately deceased [@doi:10/ggnxb3].
```
Totally shaking things up! Does this help?

File: content/20.treatments.md (hunk 9 above)
Comment (rando2) [reply content/20.treatments.md:682]: It doesn't actually! Maybe this is better:
```suggestion
They found that TCZ administration was associated with more similar rates of survivorship in patients with severe versus nonsevere COVID-19 at intake, defined based on the amount of supplemental oxygen needed.
They therefore proposed that their algorithm was able to identify patients presenting with or likely to develop CRS as good candidates for TCZ.
```

**Comment by ajlee21** [comment]
@rando2 thank you for making all those edits. I think there are just 2 very minor comments left if you want to take a look. Then I'd say its good to merge.

File: content/02.introduction.md (hunk 4 above)
Comment (rando2) [reply content/02.introduction.md:155]:
```suggestion
In this way interleukins regulate both the pro- and anti-inflammatory responses.
<!-- Note: make sure these terms are defined!-->
```

File: content/20.treatments.md (hunk 6 above)
Comment (ajlee21) [reply content/20.treatments.md:624]: I like this A LOT better!

File: content/02.introduction.md [hunk 12]
```diff
@@ -147,6 +147,21 @@ After a virus enters into a host cell, its antigen is presented by major histoco
...
+In addition, IL-6 also induces the differentiation of naïve CD4+ T cells into effector T-cell subsets [@doi:10.1158/2326-6066.CIR-14-0022].
+In this way interleukins regulate both the pro- and anti-inflammatory responses.
+<!-- Note: make sure these terms are defined!-->
+In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system, ensuring homeostasis.
+The introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-α [@doi:10.1016/j.virusres.2007.02.007], and deceased SARS-CoV patients were found to have intermediate levels of IL-6, IL-1𝛽, and TNF-α expressed in a number of ACE2-expressing cell types sampled from the lung and bronchial tissues during autopsy [@doi:10.1002/path.2067].
+However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor α (TNF-α) and IL-1𝛽 decreased with the onset of ARDS [@doi:10.1164/ajrccm.164.10.2104013].
```
Comment (rando2) [inline content/02.introduction.md:159]:
```suggestion
However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor α (TNF-α) and IL-1𝛽 decreased with the onset of ARDS [@doi:10.1164/ajrccm.164.10.2104013].
<!--

Just to clarify, is the point here that SARS patients have intermediate levels of IL-6 but other respiratory conditions, like ARDS have higher levels of IL-6?

As a non-immunologist, my high level understanding of what is written is: IL-6 plays a role in regulating the immune response to infection. And different IL-6 levels are found depending on the disease. This IL-6 cytokine causes a large immune response that damages cells and is fatal. - @ajlee21 
-->
```
